engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js

```diff
@@ -31,6 +31,10 @@ angular.module('Bastion.errata').controller('ErrataController',
         $scope.table = nutupane.table;
         $scope.removeRow = nutupane.removeRow;
 
+        Erratum.queryUnpaged({'organization_id': CurrentOrganization}, function (result) {
+            $scope.errataCount = result.total;
+        });
+
```

Comment:
I suppose I could add yet another parameter or an API to just return the count of the errata.  I was wanting to avoid that because it seems silly.

Any other suggestions?